src/groundlight/client.py

```diff
@@ -408,35 +408,38 @@ def submit_image_query(  # noqa: PLR0913 # pylint: disable=too-many-arguments, t
         :param detector: the Detector object, or string id of a detector like `det_12345`
 
         :param image: The image, in several possible formats:
+
           - filename (string) of a jpeg file
           - byte array or BytesIO or BufferedReader with jpeg bytes
-          - numpy array with values 0-255 and dimensions (H,W,3) in BGR order
-            (Note OpenCV uses BGR not RGB. `img[:, :, ::-1]` will reverse the channels)
-          - PIL Image: Any binary format must be JPEG-encoded already.
-            Any pixel format will get converted to JPEG at high quality before sending to service.
+          - numpy array with values 0-255 and dimensions (H,W,3) in BGR order (Note OpenCV uses BGR
+            not RGB. ``img[:, :, ::-1]`` will reverse the channels)
+          - PIL Image: Any binary format must be JPEG-encoded already. Any pixel format will get
+            converted to JPEG at high quality before sending to service.
 
-        :param wait: How long to poll (in seconds) for a confident answer. This is a client-side timeout.
+        :param wait: How long to poll (in seconds) for a confident answer. This is a client-side
+            timeout.
 
-        :param patience_time: How long to wait (in seconds) for a confident answer for this image query.
-            The longer the patience_time, the more likely Groundlight will arrive at a confident answer.
-            Within patience_time, Groundlight will update ML predictions based on stronger findings,
-            and, additionally, Groundlight will prioritize human review of the image query if necessary.
-            This is a soft server-side timeout. If not set, use the detector's patience_time.
+        :param patience_time: How long to wait (in seconds) for a confident answer for this image
```

Comment:
I'm using the VSCode extension ReWrap to help make the line lengths of the docstrings more balanced. Most of the docstring changes are ReWrap